cli/flox-watchdog/src/logger.rs

```diff
@@ -11,6 +11,7 @@ use tracing_subscriber::util::SubscriberInitExt;
 use tracing_subscriber::{EnvFilter, Layer};
 
 const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(3600);
+const WATCHDOG_GC_INTERVAL: Duration = Duration::from_secs(3600);
```

Comment:
nit: seems like we should make this a day since we keep files for a number of days